chainguard-dev/edu
# updating IAM docs to reflect new nomenclature

## Type of change

This PR updates several more of our IAM-related docs to use the new nomenclature around our IAM structures ("groups" ==> "organizations").

### What should this PR do?

resolves https://github.com/chainguard-dev/internal/issues/3801

### Why are we making this change?

I had been holding off from updating these docs until the Terraform provider itself was updated, but that work has been moved to the backlog for now. I'm updating these docs in an attempt to minimize confusion between them and the rest of our IAM-related docs.

### What are the acceptance criteria? 



### How should this PR be tested?

Review these changes to make sure that they all sound okay. Tech test not necessary for these.

Related: I added this note to the Terraform provider overview:

```
> **Note:** Chainguard organizations were previously called "groups," with a "root group" representing what is now referred to as an organization and "subgroups" referring to folders. As of this writing, the Chainguard Terraform provider still refers to "groups" instead of organizations. To align with our other [IAM resources](/chainguard/administration/iam-organizations/), this document uses the new nomenclature wherever possible.
```

Would this be helpful in the other docs? or would it be overkill to add it to each one?
